library/ecp_invasive.h

```diff
@@ -40,8 +40,33 @@ typedef enum {
     MBEDTLS_ECP_MOD_SCALAR
 } mbedtls_ecp_modulus_type;
 
+/* Requred macros for ECP split.
+ * If MBEDTLS_ECP_WITH_MPI_UINT is defined the new bignum interface is used.
+ */
+#ifndef MBEDTLS_ECP_WITH_MPI_UINT
+
+/* Because of `check_names.py` a define is needed for every macro. */
```

Comment:
Can I suggest

```
/* Provide a commented-out definition so that `check_names.py` knows that
 * it's not a typo.
 * MBEDTLS_ECP_WITH_MPI_UINT must not be defined within config files, but
 * only on the command line, as otherwise both ECP implementations will be
 * built.
 */
```